src/main/java/edu/wpi/first/networktables/NetworkTable.java

```diff
@@ -28,6 +30,92 @@
   private final String pathWithSep;
   private final NetworkTableInstance inst;
 
+  /**
+   * Gets the "base name" of a key. For example, "/foo/bar" becomes "bar".
+   * @param key key
+   * @return base name
+   */
+  public static String basenameKey(String key) {
```

Comment:
Is it worth mentioning that a key with a trailing slash will output an empty string `""`?